compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -223,6 +223,12 @@ getMatmulLoweringConfigAndWorkgroupSize(SmallVector<int64_t> bounds,
     }
   }
 
+  for (auto batchDim : contractionDims.batch) {
```

Comment:
```suggestion
  for (int64_t batchDim : contractionDims.batch) {
```
The type is not obvious based on the surrounding context (RHS), see: https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable